File: data/src/main/java/com/github/gmetal/data/entity/ProductionCountryEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class ProductionCountryEntity {

    @JsonField
    public String iso31661;
    @JsonField
    public String name;
}

File: data/src/main/java/com/github/gmetal/data/entity/ProductionCountryEntity.kt
package com.github.gmetal.data.entity

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class ProductionCountryEntity {

    @JsonField
    var iso31661: String? = null
    @JsonField
    var name: String? = null
}
